modules/templates/static.go

```diff
@@ -26,6 +28,11 @@ var (
 	bodyTemplates    = template.New("")
 )
 
+// GlobalModTime provide a gloabl mod time for embedded asset files
+func GlobalModTime(filename string) time.Time {
+	return timeutil.GetExecutableModTime()
```

Comment:
wonder if we could just use time of git tag

Reply:
Ah, we can get the external time via building so that we can read that time in Makefile.